### ISSUE TITLE
Domain is undefined for edit command

### ISSUE THREAD

**Issue by sorvis** [danielpigott/cloudflare-cli#85]
This works

```bash
cfcli -k $API_KEY -d review-test.com  edit testrecord.review-test.com 123.123.123.123
```

This fails because the domain is undefined when entering the find function
```bash
cfcli -k $API_KEY  edit testrecord.review-test.com 123.123.123.123
```

**Comment by LukeThoma5** [comment]
We've been unable to edit or delete a dns record, because it is passing undefined as the zone, assumedly because record.zone_id has been removed from the api response.

**Comment by christensenjairus** [comment]
I'm also having this issue, though I'm mainly concerned about the `cfcli rm` command. Seems the zone is mistakenly "undefined"

```bash
❯ cfcli rm <subdomain.domain>
Error response received: Could not route to /client/v4/zones/undefined/dns_records/<uuid>, perhaps your object identifier is invalid?
```

**Comment by Talv** [comment]
> We've been unable to edit or delete a dns record, because it is passing undefined as the zone, assumedly because record.zone_id has been removed from the api response.

Yes, the reasons are on the CF API slightly changing its behavior - it also affected my other scripts. Should have probably start using official https://github.com/cloudflare/cloudflare-typescript moving forward...

---

Anyhow, I just wanted to highlight this fix from you, to people alike me who still relay on this tool to this day.

https://github.com/LukeThoma5/cloudflare-cli/commit/25c2731ca7f6199bd029d91006d01b656597d057

Thanks @LukeThoma5 !

You should consider submitting it as PR, so it's easier to find.

**Comment by Talv** [comment]
Also this is unrelated to OP's issue.

> This works
> 
> cfcli -k $API_KEY -d review-test.com  edit testrecord.review-test.com 123.123.123.123
> This fails because the domain is undefined when entering the find function
> 
> cfcli -k $API_KEY  edit testrecord.review-test.com 123.123.123.123

In this case, you're expected to provide `CF_API_DOMAIN` as env variable. Or use the config file, but I never used it, so unsure how reliable it is.

**Comment by danielpigott** [comment]
This should be working now (better late than never)
